src/taipy/gui_core/GuiCoreLib.py

```diff
@@ -451,6 +509,28 @@ class _GuiCore(ElementLibrary):
                 "type": ElementProperty(PropertyType.inner, __DATANODE_ADAPTER),
             },
         ),
+        "job_selector": Element(
+            "value",
+            {
+                "id": ElementProperty(PropertyType.string),
+                "class_name": ElementProperty(PropertyType.dynamic_string),
+                "value": ElementProperty(PropertyType.lov_value),
+                "display_job_id": ElementProperty(PropertyType.boolean, True),
```

Comment:
I... think we have user the 'show_' prefix for these sorts of things in 'scenario'.
Wouldn't it be better?